docs/snippets/install-func.md

```diff
@@ -0,0 +1,84 @@
+<!-- Snippet used in the following topics:
+- /docs/functions/install-func.md
+- /docs/getting-started/quickstart-install.md
+- docs/install/quickstart-install.md
+-->
+
+## Install the Knative Functions CLI
+
+The Knative Functions CLI (`func`) provides an interface for creating, configuring, building and deploying Knative Functions without the need to create or modify YAML files directly. The `func` CLI can be installed as a standalone binary or as a plugin to the Knative CLI (`kn`). If you already have `kn` installed via [Homebrew](https://brew.sh){target=_blank}, when you install the `func` CLI using [Homebrew](https://brew.sh){target=_blank}, it is automatically recognized as a plugin to `kn` and can be referenced as `kn func` or `func` interchangeably.
+
+
+Do one of the following:
+
+=== "Using Homebrew"
+
+    - To install `func` using [Homebrew](https://brew.sh){target=_blank}, run the command (Use `brew upgrade` instead if you are upgrading from a previous version):
+
+        ```bash
+        brew tap knative-sandbox/kn-plugins
+        brew install func
+        ```
+
+=== "Using a binary"
+
+    You can install `func` by downloading the executable binary for your system and placing it in the system path.
+
+    1. Download the binary for your system from the [`func` release page](https://github.com/knative-sandbox/kn-plugin-func/releases){target=_blank}.
+
+    1. Rename the binary to `func` and make it executable by running the commands:
+
+        ```bash
+        mv <path-to-binary-file> func
+        chmod +x func
+        ```
+
+        Where `<path-to-binary-file>` is the path to the binary file you downloaded in the previous step, for example, `func_darwin_amd64` or `func_linux_amd64`.
+
+    1. Move the executable binary file to a directory on your PATH by running the command:
+
+        ```bash
+        mv func /usr/local/bin
+        ```
+
+    1. Verify that the CLI is working by running the command:
+
+        ```bash
+        func version
+        ```
+
+=== "Using Go"
+
+    1. Check out the `func` client repository:
+
+          ```bash
+          git clone https://github.com/knative-sandbox/kn-plugin-func.git func
+          cd func/
+          ```
+
+    1. Build an executable binary:
+
+          ```bash
+          make
+          ```
```

Comment:
Maybe `go install knative.dev/func@latest`? But when I try that I get 

```
❯ go install knative.dev/func@latest
go: downloading knative.dev/func v0.34.1
go: knative.dev/func@latest (in knative.dev/func@v0.34.1):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

Reply:
This should indeed work.
Referenced in a [new issue](https://github.com/knative/func/issues/1308)